ego4d/internal/standard_metadata_v10/component_type.csv

```diff
@@ -0,0 +1,8 @@
+component_type_id,name

```

Comment:
we probably want to have these files in S3 rather than locally - we can discuss this further